tamr_toolbox/enrichment/api_client/google_address_validate.py

```diff
@@ -0,0 +1,136 @@
+import logging
+import os
+from datetime import datetime, timedelta
+from typing import Optional
+
+from tamr_toolbox.enrichment.address_mapping import AddressValidationMapping
+from tamr_toolbox.models.data_type import JsonDict
+
+# Building our documentation requires access to all dependencies, including optional ones
+# This environments variable is set automatically when `invoke docs` is used
+BUILDING_DOCS = os.environ.get("TAMR_TOOLBOX_DOCS") == "1"
+if BUILDING_DOCS:
+    # Import relevant optional dependencies
+    import googlemaps
+
+
+LOGGER = logging.getLogger(__name__)
+_ADDRESSVALIDATION_BASE_URL = "https://addressvalidation.googleapis.com"
+
+
+def get_maps_client() -> "googlemaps.Client":
+    """Get GoogleMaps client using the environment variable 'GOOGLEMAPS_API_KEY'
```

Comment:
Does this show up in the docs as the name that is required to be set when running address validation?